src/Symfony/Component/Process/Tests/PhpProcessTest.php

```diff
@@ -26,4 +27,27 @@ public function testNonBlockingWorks()
         $process->wait();
         $this->assertEquals($expected, $process->getOutput());
     }
+
+    public function testCommandLine()
+    {
+        if (defined('PHP_BINARY')) {
+            $this->markTestSkipped('The PHP binary is easily available as of PHP 5.4');
```

Comment:
Maybe not, but [ExecutableFinderTest.php](https://github.com/symfony/symfony/blob/2.3/src/Symfony/Component/Process/Tests/ExecutableFinderTest.php#L39) have this statements.